routing/index_graph_starter.cpp

```diff
@@ -4,6 +4,18 @@
 
 namespace routing
 {
+namespace
+{
+int Sign(uint32_t a, uint32_t b)
+{
+  if (a < b)
+    return 1;
```

Comment:
See, that's why it is counterintuitive: we expect different things.
As far as I can see from how you use it, no need for a better name, just add a comment.